cmd/corim-store/cmd/corim.go

```diff
@@ -0,0 +1,205 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"slices"
+
+	"github.com/spf13/cobra"
+	"github.com/veraison/corim-store/pkg/store"
+	"github.com/veraison/corim/corim"
+)
+
+var corimCmd = &cobra.Command{
+	Use:   "corim",
+	Short: "CoRIM-related operations.",
+
```

Comment:
is there a plan to include a long description, as the short description seems in-adequate to me, likeL
"CoRIM-related operations"..????